Source/AtomicEditor/Editors/JSResourceEditor.cpp

```diff
@@ -202,6 +203,7 @@ bool JSResourceEditor::OnEvent(const TBWidgetEvent &ev)
             map["ForceSuperDown"] = true;
             #else
             map[KeyUp::P_QUALIFIERS] = QUAL_SHIFT | QUAL_CTRL;
+			webClient_->SendFocusEvent();
             #endif
```

Comment:
White spaceeeeeeee 🚀

Guessing VS code editor configured for tabs, that has bitten me so many times

![tabsspaces](https://cloud.githubusercontent.com/assets/376203/20550700/440b2596-b0ed-11e6-9ed7-700819974cad.png)